src/pg/Db.js

```diff
@@ -34,9 +34,16 @@ export default class Db {
   }
 
   async query(sql) {
-    sql.rowMode = 'array';
     log('Making SQL query: ' + sql.text, sql.values);
     try {
+      sql.types = {
+        getTypeParser: (oid, format) => {
+          if (oid === types.builtins.INT8) {
+            return (value) => parseInt(value, 10);
+          }
+          return types.getTypeParser(oid, format);
+        },
+      };
```

Comment:
This is required because the result of converting a posgres INT8 value into JSON depends on who is doing the converting.
- The postgres server's to_json, json_build_object etc. functions produce a JSON number
- The node-pg library produces a string, to prevent truncation (INT8 has 64 bytes, but JS numbers can only represent integers accurately up to 53 bits)

Here we're overriding the node-pg library's type parser to behave like Postgres itself.